pkg/cloud/vsphere/services/metadata/constants.go

```diff
@@ -16,7 +16,11 @@ limitations under the License.
 
 package metadata
 
-const format = `
+const basic = `
```

Comment:
Hi @moshloop,

Why the change here? Whether it's ESXi or vCenter, the cloud-init bits should be the same on the inside of the guest, should they not?